docs/resources/aws_iam_managed_policies.md

```diff
@@ -0,0 +1,72 @@
+---
+title: About the aws_iam_managed_policies Resource
+platform: aws
+---
+
+# aws\_iam\_policies
+
+Use the `aws_iam_managed_policies` InSpec audit resource to test properties of a collection of AWS IAM Policies.
+
+## Syntax
+
+`aws_iam_managed_policies` Resource returns a collection of IAM Policies and allows testing of that collection.
```

Comment:
`aws_iam_managed_policies` resource returns a collection of IAM Policies and allows testing of that collection.